src/WebJobs.Script.WebHost/Management/VirtualFileSystem.cs

```diff
@@ -321,7 +321,7 @@ protected Task<HttpResponseMessage> CreateDirectoryPutResponse(HttpRequest reque
             {
                 info.Create();
             }
-            catch (IOException ex)
```

Comment:
Created https://github.com/Azure/azure-functions-host/issues/2662 to track the todos in this file.